e2e/tests/visual/controlledClock.visual.spec.js

```diff
@@ -21,36 +21,55 @@
  *****************************************************************************/
 
 /*
-Collection of Visual Tests set to run in a default context. The tests within this suite
-are only meant to run against openmct's app.js started by `npm run start` within the
-`./e2e/playwright-visual.config.js` file.
-
+Collection of Visual Tests set to run with browser clock manipulate made possible with the
+clockOptions plugin fixture.
 */
 
-const { test, expect } = require('../../pluginFixtures');
+const { test } = require('../../pluginFixtures');
 const percySnapshot = require('@percy/playwright');
 
-test.describe('Visual - Controlled Clock @localStorage', () => {
+test.describe('Visual - Controlled Clock', () => {
+    test.beforeEach(async ({ page }) => {
+        //Go to baseURL and Hide Tree
+        await page.goto('./#/browse/mine?hideTree=true', { waitUntil: 'networkidle' });
+    });
     test.use({
-        storageState: './e2e/test-data/VisualTestData_storage.json',
         clockOptions: {
-            now: 0, //Set browser clock to UNIX Epoch
             shouldAdvanceTime: false //Don't advance the clock
         }
     });
 
-    test('Overlay Plot Loading Indicator @localStorage', async ({ page, theme }) => {
-        // Go to baseURL
-        await page.goto('./#/browse/mine?hideTree=true', { waitUntil: 'networkidle' });
+    test('Visual - Time Conductor start time is less than end time', async ({ page, theme }) => {
+        const year = new Date().getFullYear();
+
+        let startDate = 'xxxx-01-01 01:00:00.000Z';
```

Comment:
should we use [setFullYear](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setFullYear) here instead?